config/clusters/openscapes/cluster.yaml

```diff
@@ -1,5 +1,6 @@
 name: openscapes
 provider: aws
+account: 2i2cAdministrator
```

Comment:
@damianavila, you once shared these credentials with me. Are these still stored encrypted somewhere?